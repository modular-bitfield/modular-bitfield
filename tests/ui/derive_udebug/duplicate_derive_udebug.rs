use modular_bitfield::prelude::*;

#[bitfield]
#[derive(uDebug)] #[derive(uDebug)]
pub struct SignedInt {
    sign: bool,
    value: B31,
}

fn main() {}
