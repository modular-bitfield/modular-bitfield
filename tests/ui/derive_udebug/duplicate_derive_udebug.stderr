error: encountered duplicate `#[derive(uDebug)]` parameter
 --> tests/ui/derive_udebug/duplicate_derive_udebug.rs:4:28
  |
4 | #[derive(uDebug)] #[derive(uDebug)]
  |                            ^^^^^^

error: previous `#[derive(uDebug)]` parameter here
 --> tests/ui/derive_udebug/duplicate_derive_udebug.rs:4:10
  |
4 | #[derive(uDebug)] #[derive(uDebug)]
  |          ^^^^^^
